format.sh

```diff
@@ -25,8 +34,11 @@ find . -not -path "./third_party/**" \
   | xargs clang-format --verbose -style=google -i
 
 # Run buildifier (Bazel file formatter).
-BUILDIFIER="buildifier-$PLATFORM"
-wget "https://github.com/bazelbuild/buildtools/releases/download/v7.3.1/$BUILDIFIER"
-mv $BUILDIFIER buildifier
-chmod +x buildifier
+if [ ! -e buildifier ]
+then
+    BUILDIFIER="buildifier-$PLATFORM"
+    wget "https://github.com/bazelbuild/buildtools/releases/download/v7.3.1/$BUILDIFIER"
```

Comment:
Shall we use the latest version, i.e. `https://github.com/bazelbuild/buildtools/releases/download/v8.0.3/$BUILDIFIER`, while we're at it?

Reply:
I will try another commit that makes this change, and see what happens.

Reply:
Commit 3 updates the buildifier version to 8.0.3.  Running it caused 5 .bazel files to be updated, and those changes are also in commit 3.